### ISSUE TITLE
EventSource constantly losing connection

### ISSUE THREAD

**Issue by vacovnik** [fanout/django-eventstream#7]
I was trying out ``django-eventstream`` on a raspberry and installed ``pushpin`` and all it's dependencies directly from ppa.

Pushpin works if I use ``curl`` or ``pushpin-publish``, but with ``chat example``, I can not get it to work.
I just constantly get 

    *** connected
    *** connection lost, reconnecting...

messages.

**Comment by vacovnik** [comment]
Hi, 
Thank you to for a fast response!

The thing is that I am running raspbian lite with no display, so I am accessing a page from another computer. As you mentioned that, I went through all the steps again and noticed that `--route` flag overrides routes file, where I still had `* test` line. When I deleted it, application started to work.

The point is that django app I am trying to integrate django-eventstream in is running on a windows machine, and I would like to use raspberry just for pushpin. So if I want to achieve that, I must direct my browser to raspberry ip on routed port?

**Comment by jkarneges** [comment]
Glad you got it working. :)

> The point is that django app I am trying to integrate django-eventstream in is running on a windows machine, and I would like to use raspberry just for pushpin. So if I want to achieve that, I must direct my browser to raspberry ip on routed port?

Technically only the events endpoint needs to be accessed through Pushpin. You could load the website directly from the Windows machine if you want, as long as your JS code passes a full URL to the EventSource constructor to connect to the Pi for events.

**Comment by vacovnik** [comment]
I tried what you propose at the very beginning and didn't work, because of the previously stated error. I had some trouble figuring out the right  address, and came nowhere, so I decided to do it the other way. 

At the end the whole server will run on Pi, so I think it is better as it is. But now - for developing purposes - it's better to  code on computer with IDE than on headless machine via terminal...:)

Thank you very much for all the help and clarification.
